src/EnergyPlus/EvaporativeFluidCoolers.cc

```diff
@@ -268,37 +270,30 @@ namespace EvaporativeFluidCoolers {
 
             //   fluid bypass for single speed evaporative fluid cooler
             if (state.dataIPShortCut->lAlphaFieldBlanks(6) || AlphArray(6).empty()) {
-                thisEFC.CapacityControl = 0; // FanCycling
+                thisEFC.capacityControl = CapacityControl::FanCycling; // FanCycling
             } else {
-                {
-                    auto const SELECT_CASE_var(UtilityRoutines::MakeUPPERCase(AlphArray(6)));
-                    if (SELECT_CASE_var == "FANCYCLING") {
-                        thisEFC.CapacityControl = 0;
-                    } else if (SELECT_CASE_var == "FLUIDBYPASS") {
-                        thisEFC.CapacityControl = 1;
-                    } else {
-                        thisEFC.CapacityControl = 0;
-                        ShowWarningError(state,
-                                         state.dataIPShortCut->cCurrentModuleObject + ", \"" + thisEFC.Name +
-                                             "\" The Capacity Control is not specified correctly. The default Fan Cycling is used.");
-                    }
+                thisEFC.capacityControl =
+                    static_cast<CapacityControl>(getEnumerationValue(controlNamesUC, UtilityRoutines::MakeUPPERCase(AlphArray(6))));
+                if (thisEFC.capacityControl == CapacityControl::Invalid) {
+                    thisEFC.capacityControl = CapacityControl::FanCycling;
+                    ShowWarningError(state,
+                                     state.dataIPShortCut->cCurrentModuleObject + ", \"" + thisEFC.Name +
+                                         "\" The Capacity Control is not specified correctly. The default Fan Cycling is used.");
                 }
             }
 
             thisEFC.SizFac = NumArray(12); //  N11  \field Sizing Factor
             if (thisEFC.SizFac <= 0.0) thisEFC.SizFac = 1.0;
 
-            // begin water use and systems get input
-            if (UtilityRoutines::SameString(AlphArray(7), "LossFactor")) {
-                thisEFC.EvapLossMode = EvapLoss::ByUserFactor;
-            } else if (UtilityRoutines::SameString(AlphArray(7), "SaturatedExit")) {
-                thisEFC.EvapLossMode = EvapLoss::ByMoistTheory;
-            } else if (AlphArray(7).empty()) {
+            if (AlphArray(7).empty()) {
                 thisEFC.EvapLossMode = EvapLoss::ByMoistTheory;
             } else {
-                ShowSevereError(state, "Invalid, " + state.dataIPShortCut->cAlphaFieldNames(7) + " = " + AlphArray(7));
-                ShowContinueError(state, "Entered in " + state.dataIPShortCut->cCurrentModuleObject + " = " + AlphArray(1));
-                ErrorsFound = true;
+                thisEFC.EvapLossMode = static_cast<EvapLoss>(getEnumerationValue(evapLossNamesUC, UtilityRoutines::MakeUPPERCase(AlphArray(7))));
```

Comment:
Standard new approach of using getEnumerationValue and checking for invalid...carrying on...